lib/src/note/positioned_note.dart

```diff
@@ -136,15 +138,14 @@ final class PositionedNote extends Note {
 
   @override
   bool operator ==(Object other) =>
-      super == other && other is PositionedNote && octave == other.octave;
+      other is PositionedNote && note == other.note && octave == other.octave;
 
   @override
   int get hashCode => Object.hash(super.hashCode, octave);
```

Comment:
This deprecated `hashCode` was left behind, as no exhaustive tests existed to catch a bad implementation there.

Fixed in #155.